river/command/map.zig

```diff
@@ -165,31 +163,27 @@ fn pointerMappingExists(
     return null;
 }
 
-fn parseEventCode(allocator: mem.Allocator, name: [:0]const u8, out: *?[]const u8) !u32 {
+fn parseEventCode(name: [:0]const u8, out: *?[]const u8) !u32 {
     const event_code = c.libevdev_event_code_from_name(c.EV_KEY, name);
     if (event_code < 1) {
-        out.* = try fmt.allocPrint(allocator, "unknown button {s}", .{name});
+        out.* = try fmt.allocPrint(util.gpa, "unknown button {s}", .{name});
         return Error.Other;
     }
 
     return @intCast(u32, event_code);
 }
 
-fn parseKeysym(allocator: mem.Allocator, name: [:0]const u8, out: *?[]const u8) !xkb.Keysym {
+fn parseKeysym(name: [:0]const u8, out: *?[]const u8) !xkb.Keysym {
     const keysym = xkb.Keysym.fromName(name, .case_insensitive);
     if (keysym == .NoSymbol) {
-        out.* = try fmt.allocPrint(allocator, "invalid keysym '{s}'", .{name});
+        out.* = try fmt.allocPrint(util.gpa, "invalid keysym '{s}'", .{name});
         return Error.Other;
     }
     return keysym;
 }
 
-fn parseModifiers(
-    allocator: mem.Allocator,
-    modifiers_str: []const u8,
-    out: *?[]const u8,
-) !wlr.Keyboard.ModifierMask {
-    var it = mem.split(u8, modifiers_str, "+");
+fn parseModifiers(modifiers_str: []const u8, out: *?[]const u8) !wlr.Keyboard.ModifierMask {
+    var it = std.mem.split(u8, modifiers_str, "+");
```

Comment:
This change seems to be unintentional.
```suggestion
    var it = mem.split(u8, modifiers_str, "+");
```

Reply:
ah yeah I start before the last commit was merged in master, I know I would messed something when rebasing. Fixed